thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datasource/cache/DatasetsCache.java

```diff
@@ -5,56 +5,56 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 import org.apache.commons.collections.CollectionUtils;
-import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import jersey.repackaged.com.google.common.collect.Lists;
-
 import com.linkedin.thirdeye.common.ThirdEyeConfiguration;
 import com.linkedin.thirdeye.datalayer.bao.DatasetConfigManager;
 import com.linkedin.thirdeye.datalayer.dto.DatasetConfigDTO;
 
-public class CollectionsCache {
+public class DatasetsCache {
```

Comment:
This is not the dataset config cache. This simply contained the list of collections